README.md

```diff
@@ -26,3 +26,15 @@ Interface to use [react-with-styles][react-with-styles] with [Aphrodite][aphrodi
 
 [react-with-styles]: https://github.com/airbnb/react-with-styles
 [aphrodite]: https://github.com/khan/aphrodite
+
+## Import
+
+```
```

Comment:
It would be nice to add `js` to the end of these lines to enable syntax highlighting